### PR TITLE
move to manifest v2

### PR DESCRIPTION
**Pull request by rberrelleza**
we need a sample with v2 + dependencies. I took @teresaromero branch, but change it to use MongoDB so we keep everything in the same deployment technology, so we can make this demo just about multiple repos.

### CODE REVIEW

**Comment by rberrelleza** [comment]
https://github.com/okteto/movies-frontend/pull/1 is the PR for the other service

File: okteto.yml [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+  - name: Deploy MongoDB
+    command:  helm upgrade --install mongodb bitnami/mongodb -f mongodb/values.yml --version 7.2.11
+  - name: Deploy API
+    command: helm upgrade --install movies-api chart --set image=${OKTETO_BUILD_API_IMAGE}
+
+dev:
+  api:
+    command: bash
+    sync:
+      - .:/src
```
Comment (pchico83) [inline okteto.yml]: You are missing:
```
forward:
  - 9229:9229
  - 27017:mongodb:27017
```

File: okteto.yml [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
-sync:
-  - .:/src
-forward:
-  - 9229:9229
-  - 27017:mongodb:27017
+build:
+  api:
+    context: .
+
+dependencies:
```
Comment (pchico83) [inline okteto.yml:6]: you can also do:
```
dependencies:
  - https://github.com/okteto/movies-frontend
```

File: okteto.yml (hunk 1 above)
Comment (rberrelleza) [reply okteto.yml]: updated!

File: okteto.yml (hunk 2 above)
Comment (rberrelleza) [reply okteto.yml:6]: updated!

**Review by pchico83 — APPROVED** [review]

**Pull request merged**